mobile/src/main/java/ch/epfl/sdp/mobile/state/StatefulGameScreen.kt

```diff
@@ -99,3 +102,30 @@ private fun StatefulPromoteDialog(
     )
   }
 }
+
+/**
+ * A composable which displays a snackbar to show the [SpeechRecognizerState.currentError] when it's
+ * modified
+ *
+ * @param state the [SpeechRecognizerState] which backs this dialog.
+ * @param snackbarHostState the [SnackbarHostState] used to display some results.
+ */
+@Composable
+private fun StatefulSnackBar(
+    state: SpeechRecognizerState,
+    snackbarHostState: SnackbarHostState,
+) {
```

Comment:
Question: Should this Composable allow a modifier? (That is, do any of it's components like the LaunchedEffect allow themselves a composable)

Reply:
This function didn't contain a composable component. It's only used to call `showSnackbar` on an existing composable. 

We don't need a modifier here, because we don't set the "visual" of the component